4337/test/eip4337/EIP4337Module.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import { Signer } from 'ethers'
+import { getTestSafe, getSimple4337Module, getEntryPoint } from '../utils/setup'
+import { buildSignatureBytes, signHash, logGas } from '../../src/utils/execution'
+import {
+  buildSafeUserOp,
+  calculateSafeOperationHash,
+  buildUserOperationFromSafeUserOperation,
+  buildSafeUserOpTransaction,
+  signSafeOp,
+} from '../../src/utils/userOp'
+import { chainId } from '../utils/encoding'
+
+describe('Simple4337Module', async () => {
```

Comment:
```suggestion
describe('Simple4337Module', () => {
```
This doesn't need to be async anymore

Reply:
Nice catch - can probably be removed everywhere actually, now that the `let [user] = await ethers.getSigners()` moved to the async `setup` function.

Reply:
300395be039b3963834216463be2cab93705ef1d